src/app/lab/lab.component.ts

```diff
@@ -2,21 +2,19 @@
  * Climate Change Lab
  * App Component
  */
-import { Component, OnInit, OnDestroy, ViewEncapsulation } from '@angular/core';
+import { Component, OnInit, OnDestroy, ViewEncapsulation, Input } from '@angular/core';
+import { Router, ActivatedRoute }     from '@angular/router';
+
 import { SidebarComponent } from '../sidebar/sidebar.component';
-import { ChartService } from '../services/chart.service';
-import { ClimateModelService } from '../services/climate-model.service';
-import { ScenarioService } from '../services/scenario.service';
+import { ScenarioDropdownComponent } from './dropdowns/scenario-dropdown.component';
```

Comment:
No need to import these here, they're available already because they're defined in the app module.